lab_1_classify_by_unigrams/main.py

```diff
@@ -88,6 +169,20 @@ def collect_profiles(paths_to_profiles: list) -> list[dict[str, str | dict[str,
     :paths_to_profiles: a list of strings to the profiles
     :return: a list of loaded profiles
     """
+    if not isinstance(paths_to_profiles, list):
+        return None
+    collect_profs = []
+    for prof in paths_to_profiles:
+        if not load_profile(prof):
+            return None
+        load_prof = load_profile(prof)
+        if load_prof:
```

Comment:
Why do not in the same way as `load_profile` ?

Reply:
aaah

Reply:
because mypy wanted me to be sure that the variable in preprocess_rpofile was really a dict

Reply:
yaay fixed